### DISCUSSION TITLE
Feature Request: Display Latest Version of Packages on Hover in PNPM Catalog

### DISCUSSION THREAD

**Discussion by sohamnandi77** [pnpm/pnpm#8798] [General]
**Context**
Currently, the PNPM Catalog provides a great way manage packages in monorepos. However, when reviewing packages, it's often necessary to check the latest available version to determine compatibility or update status.

**Problem**
There is no direct way to know the latest version of a package in the PNPM Catalog without clicking on the package or looking it up separately. This adds extra steps when reviewing or deciding which package to use or update.

**Proposal**
Introduce a feature where the latest version of a package is displayed when hovering over its name in the catalog, similar to how it's done in `package.json`. This would:
- Improve usability by reducing the time spent checking for package updates.
- Provide a quick overview of package versions without leaving the current page or context.

**Comment by zkochan** [comment]
We don't maintain any vs code extensions to support this.

**Reply by lvqq** [reply]
Could we have an official vscode extension? I'm willing to maintain it.
